services/src/main/scala/cromwell/services/keyvalue/impl/KeyValueDatabaseAccess.scala

```diff
@@ -1,34 +1,7 @@
 package cromwell.services.keyvalue.impl
```

Comment:
Happy to TechTalk™️ this on Monday.  The choice of name may depend on whether this same store is used for call caching.